packages/flutter/test/material/choice_chip_test.dart

```diff
@@ -132,4 +132,52 @@ void main() {
     final RawChip rawChip = tester.widget(find.byType(RawChip));
     expect(rawChip.iconTheme, iconTheme);
   });
+
+  testWidgets('ChoiceChip passes showCheckMark property to RawChip', (WidgetTester tester) async {
```

Comment:
```suggestion
  testWidgets('ChoiceChip passes checkmark properties to RawChip', (WidgetTester tester) async {
```

Test both `showCheckmark` and `checkmarkColor` properties are passed to `RawChip`

Reply:
We should add the same test but use `ChipTheme` to pass these properties.

Reply:
@9oya  
Can you please duplicate this test and use `ChipTheme` to test if `showCheckmark` is passed from the theme?

Reply:
Okay, I will 🙆‍♂️

Reply:
@TahaTesser I am trying to check `ChipTheme`, but it isn't affected by `ChoiceChip's` property like `RawChip` does. You can check `class _ChoiceChipDefaultsM3 extends ChipThemeData` in `choice_chip.dart`. 
The `checkmarkColor` and `showCheckmark` properties of `ThemeData(useMaterial3: true).chipTheme` will always return null values.

Reply:
We've tested `ChoiceChip` properties which you updated here. Just do the same for theme value we're passing

Add this test below the `ChoiceChip passes iconTheme property to RawChip'` test.


```dart
  testWidgets('ChoiceChip passes showCheckmark from ChipTheme to RawChip', (WidgetTester tester) async {
    const bool showCheckmark = false;
    const Color checkmarkColor = Color(0xff0000ff);
    await tester.pumpWidget(wrapForChip(
      child: const ChipTheme(
        data: ChipThemeData(
          showCheckmark: showCheckmark,
          checkmarkColor: checkmarkColor,
        ),
        child: ChoiceChip(
          label: Text('Test'),
          selected: true,
          showCheckmark: showCheckmark,
        ),
      )));
    final RawChip rawChip = tester.widget(find.byType(RawChip));
    expect(rawChip.showCheckmark, showCheckmark);
  });
```

Reply:
![Screenshot 2023-04-19 at 15 08 21](https://user-images.githubusercontent.com/48603081/233070537-a716f842-c300-4ebf-86a1-60dd8e463caa.png)

Reply:
We don't need to test the `checkmarkColor` theme value here since we're not providing it here.

Reply:
Oh now I got your point!